strawberryfields/backends/__init__.py

```diff
@@ -26,7 +26,7 @@
     "TFBackend",
 ]
 
-supported_backends = {"base": BaseBackend, "gaussian": GaussianBackend, "fock": FockBackend}
+supported_backends = {b.short_name: b for b in (BaseBackend, GaussianBackend, FockBackend)}
```

Comment:
I guess TFBackend is not here because TF is no longer a requirement?

Reply:
Yeah, `load_backend()` takes care of `TFBackend` by importing the `tfbackend` submodule and tensorflow on-demand.